components/RNBHeader.tsx

```diff
@@ -84,46 +95,137 @@ export default function RNBHeader({ withNavigation = true }: Props) {
+        iconId: 'fr-icon-arrow-left-line',
+        linkProps: {
+          href: '/',
+        },
+        text: 'Retour au site',
+      };
+      newQuickActions.push(goBackToSiteQA);
+
+      // Title
+
+      setTitle(
         // @ts-ignore
-        onClick: (e) => {
-          handleSignout(e);
+        <>
+          Référentiel National des Bâtiments
+          <br />
+          <Badge as="span" noIcon severity="success" small>
+            Mode Édition
+          </Badge>
+        </>,
+      );
+
+      // Edition guide
+      let editionGuide = {
+        iconId: 'fr-icon-community-line',
+        linkProps: {
+          href: '/guide',
         },
-      },
-      text: 'Se déconnecter',
-    };
-  }
+        text: "Guide d'édition",
+      };
+      newQuickActions.push(editionGuide);
+    } else {
+      // Go to Edition
 
-  useEffect(() => {
-    setRedirectUrl(window.location.href);
-  }, [pathname]);
+      if (enableEditionMode) {
+        let goToEditionQA = editBtn;
+        newQuickActions.push(goToEditionQA);
+      }
 
-  const editModeToggler = <ToggleEditMode />;
+      setTitle('Référentiel National des Bâtiments');
+    }
 
-  const enableEditionMode =
-    process.env.NEXT_PUBLIC_ENABLE_EDITION_MODE === 'true';
-  const quickAccessItems = [enableEditionMode ? editModeToggler : null, logQA];
+    // Show some quick actions depending on the login status
+    if (session) {
+      // My account
+      let myAccountQA = {
+        iconId: 'ri-arrow-down-circle-line',
+        linkProps: {
+          href: '/mon-compte',
+        },
+        text: 'Mon compte',
+      };
+      newQuickActions.push(myAccountQA);
+    } else {
+      let loginQA = {
+        iconId: 'fr-icon-account-circle-line',
+        linkProps: {
+          href: '/login?redirect=' + redirectUrl,
+        },
+        text: 'Se connecter',
+      };
+      newQuickActions.push(loginQA);
+
+      let registerQA = {
+        iconId: 'fr-icon-add-circle-fill',
+        linkProps: {
+          href: '/login',
+        },
+        text: "S'inscrire",
+      };
+      newQuickActions.push(registerQA);
+    }
+
+    // Set
+    // @ts-ignore
+    setQuickActions(newQuickActions);
+  }, [pathname, session]);
 
   return (
     <>
+      <explainModal.Component title="Participer au Référentiel National des Bâtiments">
+        <Notice
+          title="Phase de test :"
+          description="le service d'édition est en phase de test en vue d'un démarrage
+            officiel fin juin 2025."
+          severity="info"
+          isClosable={false}
+          className="fr-my-4w"
+        />
+
+        <p>
+          Le Référentiel National des Bâtiments référence l&apos;ensemble des
+          bâtiments du territoire et leur attribue un identifiant unique.
+        </p>
+        <p>
+          Pour être constamment à jour, il est alimenté par de grandes bases de
+          données à la portée nationale mais aussi par les contributions de ceux
```

Comment:
```suggestion
          données à portée nationale mais aussi par les contributions de ceux
```
ou "de couverture nationale" ?